src/main/java/seedu/address/logic/commands/EditLeaveCommand.java

```diff
@@ -39,7 +38,9 @@
     public static final String MESSAGE_ALREADY_APPROVE = "This leave application is already approved.";
     public static final String MESSAGE_ALREADY_REJECTED = "This leave application is already rejected.";
     public static final String MESSAGE_INVALID_LEAVE_APPROVAL =
-            "Not authorized to approve leave application.";
+            "Not authorized to approve/reject this leave application.";
+    public static final String MESSAGE_INVALID_LEAVE_APPROVAL2 =
+            "Not authorized to approve/reject this leave application only another ADMIN can approve your leave.";
```

Comment:
The string for this can be made a little clearer.

Reply:
What do you mean CLEARER?

Reply:
What kind of admin do you mean? IT Admin? Administrator? I feel that the sentence is like a little vague.